torchtune/modules/loss/ce_chunked_output_loss.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import List
+
+import torch
+import torch.nn.functional as F
+
+
+class CEWithChunkedOutputLoss(torch.nn.Module):
+    """
+    CE with chunked outputs that saves memory by only upcasting one chunk at a time.
+
+    Since the model is trained with fp16, before running CE, we have to upcast
```

Comment:
should be this, right? Alternatively can just keep it general and say something like "16-bit precision" to cover all cases 
```suggestion
    Since the model is trained with bf16, before running CE, we have to upcast
```